### DISCUSSION TITLE
Not compatible with capsolver extension

### DISCUSSION THREAD

**Discussion by gtxg16** [ultrafunkamsterdam/undetected-chromedriver#1635] [Q&A]
So I bypass captchas with capsolver. However, if I use UC with capsolver, the solver will not work. However, it is only with normal selenium chrome driver it will work. Maybe there is something in UC blocking capsolver extension?

The problem is normal selenium will be detected by the website, even with all the things I find online that aren't saying to use UC. So what do I do?

Thx

**Comment by mir3u** [comment]
How are you loading the extension? If you try with the packed version(etx) it will not work.

**Reply by gtxg16** [reply]
no im loading it as a folder
